lib/authority_ecto/password.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Authority.Ecto.Password do
```

Comment:
perhaps this should be marked `@moduledoc false`?

Reply:
Yes, I agree with that.